props/saber_fett263_buttons.h

```diff
@@ -257,7 +331,96 @@ SaberFett263Buttons() : PropBase() {}
       case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_OFF | BUTTON_AUX):
         previous_preset();
         return true;
+
+       // Gesture Controls
+		
+       // Battle Mode
+		    
+#ifdef FETT263_BATTLE_MODE
+       case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_ON | BUTTON_AUX):
+         if (!battle_mode_) {
+           battle_mode_ = true;
+           hybrid_font.SB_Force();
+         } else {
+           battle_mode_ = false;
+           hybrid_font.SB_Force();
+         }
+         return true;
+
+       // Auto Lockup Mode
+       case EVENTID(BUTTON_NONE, EVENT_CLASH, MODE_ON):
+         if (!battle_mode_) return false;
+         clash_impact_millis_ = millis();
+         swing_blast_ = false;
+         if (!swinging_) {
+           SaberBase::SetLockup(SaberBase::LOCKUP_NORMAL);
+           swing_blast_ = false;
+           auto_lockup_on_ = true;
+           SaberBase::DoBeginLockup();
+           return true;
+	 }
+
+       case EVENTID(BUTTON_NONE, EVENT_STAB, MODE_ON):
+         if (!battle_mode_) return false;
+         clash_impact_millis_ = millis();
+         swing_blast_ = false;
+         if (!swinging_) {
+           if (fusor.angle1() < - M_PI / 4) {
+             SaberBase::SetLockup(SaberBase::LOCKUP_DRAG);
+             } else {
```

Comment:
not indented right (goes for most of the lines from here to the next case)

Reply:
not clear on this I have 2 spaces for each level below the case, are you meaning everything has to be indented further?

Reply:
every { means that indentation should increase by two spaces
every } means that indentation should decrease by two spaces

an if statement with an else, should look like:

if (.....) {
  code
} else {
  code
}

The if and the two } should always be in the same column.
Having the columns line up makes it possible to see where a block of code begins and ends easily and thus makes the code much easier to read.